### PR TITLE
Update best-practices.md

### PR DESCRIPTION
**Pull request by mmattel**
The admonition example had square brackets but you must not use them.
Would not work otherwise. Was a copy/paste mistake.

### CODE REVIEW

File: docs/best-practices.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ Asciidoctor provides five admonition style labels: `NOTE`, `TIP`, `IMPORTANT`, `
 
 An `admonition` is written in following example style: `[<label>:]` Text
 
-Example: `[NOTE:]` If you have a large ownCloud installation and have shell access, you should ...
+Example: `NOTE:` If you have a large ownCloud installation and have shell access, you should ...
```
Comment (settermjd) [inline docs/best-practices.md]: Markdown doesn't support admonitions. What about just bolding the text?

Comment (mmattel) [reply docs/best-practices.md]: This is not Markdown !
We write BP for AsciiDoc 
This example is for AsciiDoc presented in Markdown :smile:

Comment (settermjd) [reply docs/best-practices.md]: There is such a thing?

Comment (mmattel) [reply docs/best-practices.md]: Yes, believe me

**Pull request merged**